### ISSUE TITLE
[1.17.1 G9 pre33] Crashes on Linux Mint

### ISSUE THREAD

**Issue by spitemim** [sp614x/optifine#6159]
## Description of Issue
Optifine 1.17.1 G9 pre33 crashes on startup when using Linux Mint. [Error message](http://0x0.st/-J4Z.png)

## Steps to Reproduce
1. Be on Linux Mint
2. Download and install Optifine
3. Run Optifine via the Minecraft Launcher

## OptiFine Version
OptiFine HD U G9 pre33

## Installation Method
I downloaded the Optifine .jar file from optifine.net via the mirror link and opened it with `java -jar optifine.jar`.

## Other Installed Mods
I don't know if this is what you mean, but:
```
1.16.3-OptiFine_HD_U_G3
1.16.4-OptiFine_HD_U_G5
1.16.5-OptiFine_HD_U_G7
1.17.1-OptiFine_HD_U_G9_pre27
1.17.1-OptiFine_HD_U_G9_pre29
1.17.1-OptiFine_HD_U_G9_pre33
1.8.9-OptiFine_HD_U_M5
```

## Log Files/Crash Reports
[Pastebin link](https://pastebin.com/KVtYFgyd)

## F3 Debug Screenshot
n/a

## Additional Information
If anyone else is on Linux Mint/an Ubuntu based system and Optifine works for them, then it's probably just my system.

**Comment by fayer3** [comment]
already reported here: #5739

**Comment by KaiAF** [comment]
Should be fixed now, posted here: #5739
